examples/source_stream_wfs_3d.html

```diff
@@ -124,9 +124,9 @@
             view.addLayer(lyonTclBusLayer);
 
             function colorBuildings(properties) {
-                if (properties.id.indexOf('bati_remarquable') === 0) {
+                if (properties.id_json.indexOf('bati_remarquable') === 0) {
```

Comment:
Why there's already a `id` key?

Reply:
In geojson features, there is an `id` property with a value like "_bati_remarquable.107720_". There is also a `properties` property which contains an `id` like "_BATIMENT0000000240851490_". I don't know if there is an interest in keeping the "_BATIMENT0000000240851490_" id in this specific example, but I assume the good standard procedure would be to keep any duplicate keys.
In the example this result in the fact that the `id` determining the building type (i.e. "_bati_remarquable.107720_") would override the other id ("_BATIMENT0000000240851490_"), so it is suffixed.

Reply:
This made me wonder : which `id` ("_BATIMENT0000000240851490_" or "_bati_remarquable.107720_") do we want to display in the top-right information (see screenshot below) ?
![Screenshot from 2021-04-06 16-15-34](https://user-images.githubusercontent.com/73115044/113725606-c5855100-96f3-11eb-9c85-79d457e41a98.png)

Reply:
In `geojson` [specification](https://tools.ietf.org/html/rfc7946) `id` is the only member that we don't process.
If there are other members, they are `foreign members`.

we could formalise, as follows:

```js
properties: {
    id: 'a1'
    type: 'other'
    geojson: {
          id: 'bat1',
          tittle: 'geom',
    }
}

```